conversions/octal_to_binary.py

```diff
@@ -0,0 +1,49 @@
+"""
+Convert a octal value to its binary equivalent
+>>> octal_to_binary("17")
+'1111'
+>>> octal_to_binary("52523")
+'101010101010011'
+>>> octal_to_binary("532.51")
+'101011010.101001'
```

Comment:
These doctests should go inside a docstring within the function, not at the top of the file.